Integrations/Snowflake/Snowflake.yml

```diff
@@ -40,7 +40,7 @@ configuration:
   name: proxy
   required: false
   type: 8
-- display: Trust server certificate (insecure)
+- display: Trust any certificate (not secure)
```

Comment:
I thought it was agreed that we were switching back from `unsecure` (since it's improper english) to `insecure` as a convention?

Reply:
we finalised on `not secure`. See: https://github.com/demisto/etc/issues/14980